.github/workflows/continuous-monitoring.yml

```diff
@@ -0,0 +1,43 @@
+name: Continuous monitoring of distribution channels
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '0 */1 * * *'
+
+jobs:
+  pull-agent:
+    name: Pull X-Ray java agent from Maven Central and Github
+    runs-on: ubuntu-latest
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+                 
+      - name: Pull Java agent from Github
+        id: distribution-availability-github
+        run: wget https://github.com/aws/aws-xray-java-agent/releases/download/v2.8.0/xray-agent.zip
```

Comment:
We want to pull the latest version of the agent always, so we can link here instead: https://github.com/aws/aws-xray-java-agent/releases/latest/download/xray-agent.zip